src/executor/query/GetNeighborsExecutor.cpp

```diff
@@ -69,7 +69,7 @@ Status GetNeighborsExecutor::buildRequestDataSet() {
 folly::Future<Status> GetNeighborsExecutor::getNeighbors() {
     if (reqDs_.rows.empty()) {
         LOG(INFO) << "Empty input.";
-        DataSet emptyResult;
```

Comment:
Executor output result should always be a `DataSet`. what problem does the empty DataSet have?

Reply:
The GetNeighborsIter only accept the List < DataSet >